tests/baselines/reference/esDecorators-classDeclaration-exportModifier.2.errors.txt

```diff
@@ -42,4 +49,50 @@ tests/cases/conformance/esDecorators/classDeclaration/file7.ts(2,21): error TS80
                         ~~~~
 !!! error TS8038: Decorators may not appear after 'export' or 'export default' if they also appear before 'export'.
 !!! related TS1486 tests/cases/conformance/esDecorators/classDeclaration/file7.ts:2:1: Decorator used before 'export' here.
+    
+==== tests/cases/conformance/esDecorators/classDeclaration/file8.ts (0 errors) ====
+    // ok
+    @dec abstract class C8 {}
+    
+==== tests/cases/conformance/esDecorators/classDeclaration/file9.ts (0 errors) ====
+    // ok
+    @dec export abstract class C9 {}
+    
+==== tests/cases/conformance/esDecorators/classDeclaration/file10.ts (0 errors) ====
+    // ok
+    @dec export default abstract class C10 {}
+    
+==== tests/cases/conformance/esDecorators/classDeclaration/file11.ts (0 errors) ====
+    // ok
+    export @dec abstract class C11 {}
+    
+==== tests/cases/conformance/esDecorators/classDeclaration/file12.ts (0 errors) ====
+    // ok
+    export default @dec abstract class C12 {}
+    
+==== tests/cases/conformance/esDecorators/classDeclaration/file13.ts (2 errors) ====
+    // error

```

Comment:
aside - I don't like the `// ok` vs. `// error` comments because it is possible for them to change over time. I would prefer the baselines be the source of truth.

Reply:
I'm torn, because on one hand, the baselines are the source of truth, but on the other, if I didn't write the baselines, I don't have any indication as to what the original intended behavior was...

Reply:
They were, at the very least, useful to verify that I was getting errors where I expected them when running the tests initially. I can remove the comments, or make them more detailed to explain _why_ an error is expected at these locations.